src/applications/disability-benefits/686/helpers.jsx

```diff
@@ -103,3 +103,13 @@ export function isVeteran(item) {
   return relationship && relationship === '1';
 }
 
+export function authorize(isVerified) {
+  if (isVerified) {
+    // verifyDisabilityRating();
+  }
+}
+
+export const profileStatuses = { // TODO: move to platform and use in requiredLoginView
+  serverError: 'SERVER_ERROR',
+  notFound: 'NOT_FOUND'
```

Comment:
I'd expect these keys to also be SNAKE_CASE just like the values are.

Reply:
Sounds good, updated.